wake/development/core.py

```diff
@@ -1462,6 +1462,9 @@ class Chain(ABC):
 
     tx_callback: Optional[Callable[[TransactionAbc], None]]
 
+    def __deepcopy__(self, memo):
```

Comment:
Other than the data stored in the FuzzTest class would not stored,
we store random state already in data collecting phase.
chain and account information couls store in chain.snapshot().

Even tester defined outside of FuzztestClass, chain state would stored.
since those chains are added in wake.testing.core.connected_chains as global variable and Shrinking use this array.

Reply:
yeah, I think there are a few attributes missing to be stored in `snapshot`, then it should be good